tests/test_inspector.py

```diff
@@ -31,10 +34,27 @@
     check_timestamps_match_first_dimension,
 )
 from nwbinspector.testing import make_minimal_nwbfile
+from nwbinspector.tools import BACKEND_IO_CLASSES
 from nwbinspector.utils import FilePathType
 
+IO_CLASSES_TO_BACKEND = {v: k for k, v in BACKEND_IO_CLASSES.items()}
+EXPECTED_REPORTS_FOLDER_PATH = Path(__file__).parent / "expected_reports"
+
+
+def add_big_dataset_no_compression(nwbfile: NWBFile, zarr: bool = False) -> None:
+    # Zarr automatically compresses by default
+    # So to get a test case that is not compressed, forcibly disable the compressor
+    if zarr:
+        time_series = TimeSeries(
+            name="test_time_series_1",
+            data=hdmf_zarr.ZarrDataIO(np.zeros(shape=int(1.1e9 / np.dtype("float").itemsize)), compressor=False),
+            rate=1.0,
+            unit="",
+        )
+        nwbfile.add_acquisition(time_series)
```

Comment:
This one was interesting to me - it indicates that the majority of NWB Zarr files should not trigger either of these errors since disabling compression takes additional effort as shown